tests/load_FWs.py

```diff
@@ -87,36 +93,40 @@ def load_can(drive_conf, mc):
     ini_time = time.perf_counter()
     while (time.perf_counter() - ini_time) <= TIMEOUT_NEW_FW_DETECT and not detected:
         try:
-            mc.communication.connect_servo_canopen(
-                CAN_DEVICE(drive_conf["device"]),
-                drive_conf["dictionary"],
-                drive_conf["node_id"],
-                CAN_BAUDRATE(drive_conf["baudrate"]),
-                channel=drive_conf["channel"],
-            )
+            net, servo = connect_can(drive_conf)
             # Reaching this point means we are connected
             detected = True
-            logger.info("New FW detected after: {:.1f} s".format(time.perf_counter() - ini_time))
-            mc.communication.disconnect()
+            fw_version = servo.info["firmware_version"]
+            logger.info(
+                f"New FW detected ({fw_version}) after: {time.perf_counter() - ini_time:.1f} s"
+            )
+            net.disconnect_from_slave(servo)
         except Exception as e:
             # When cannot connect
             time.sleep(SLEEP_TIME_NEW_FW_DETECT)
```

Comment:
Why this sleep is needed?

Reply:
Not sure. I guess it waits to let the drive start the program.